ERROR - 2024-06-18 07:11:17 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1829): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2218): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\crud_telat\app\Database\Seeds\TipoUsuarioSeeder.php(26): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\TipoUsuarioSeeder->run()
#7 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\crud_telat\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-06-18 18:58:21 --> Error: Class "App\Models\DepartamentosModel" not found
[Method: GET, Route: empleados/new]
in APPPATH\Controllers\Empleados.php on line 39.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->new()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-06-18 20:28:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-18 20:28:58 --> mysqli_sql_exception: Table 'telat.departamentos' doesn't exist in C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', Array, NULL, 'id_tipo_usuario')
#6 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', Array, Array, 'id_tipo_usuario')
#7 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\crud_telat\app\Controllers\Empleados.php(68): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Empleados->create()
#10 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#11 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\crud_telat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-06-18 20:28:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.departamentos' doesn't exist
[Method: POST, Route: empleados]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `departamentos`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:28:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.departamentos' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `departamentos`
WHERE `id` = \'1\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `departamentos`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `departamentos`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:28:58 --> [Caused by] mysqli_sql_exception: Table 'telat.departamentos' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1
FROM `departamentos`
WHERE `id` = \'1\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `departamentos`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `departamentos`
WHERE `id` = \'1\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `departamentos`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-18 20:32:22 --> mysqli_sql_exception: Table 'telat.id_tipo_usuario' doesn't exist in C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', Array, NULL, 'id_tipo_usuario')
#6 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', Array, Array, 'id_tipo_usuario')
#7 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\crud_telat\app\Controllers\Empleados.php(68): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Empleados->create()
#10 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#11 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\crud_telat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-06-18 20:32:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.id_tipo_usuario' doesn't exist
[Method: POST, Route: empleados]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:32:22 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.id_tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:32:22 --> [Caused by] mysqli_sql_exception: Table 'telat.id_tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-18 20:32:46 --> mysqli_sql_exception: Table 'telat.tipo_usuario' doesn't exist in C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', Array, NULL, 'id_tipo_usuario')
#6 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', Array, Array, 'id_tipo_usuario')
#7 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\crud_telat\app\Controllers\Empleados.php(68): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Empleados->create()
#10 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#11 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\crud_telat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-06-18 20:32:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.tipo_usuario' doesn't exist
[Method: POST, Route: empleados]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:32:46 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:32:46 --> [Caused by] mysqli_sql_exception: Table 'telat.tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-18 20:35:43 --> mysqli_sql_exception: Table 'telat.id_tipo_usuario' doesn't exist in C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', Array, NULL, 'id_tipo_usuario')
#6 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', Array, Array, 'id_tipo_usuario')
#7 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\crud_telat\app\Controllers\Empleados.php(68): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Empleados->create()
#10 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#11 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\crud_telat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-06-18 20:35:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.id_tipo_usuario' doesn't exist
[Method: POST, Route: empleados]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:35:43 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.id_tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:35:43 --> [Caused by] mysqli_sql_exception: Table 'telat.id_tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `id_tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-18 20:38:38 --> mysqli_sql_exception: Table 'telat.tipo_usuario' doesn't exist in C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', Array, NULL, 'id_tipo_usuario')
#6 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', Array, Array, 'id_tipo_usuario')
#7 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\crud_telat\app\Controllers\Empleados.php(68): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Empleados->create()
#10 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#11 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\crud_telat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-06-18 20:38:38 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.tipo_usuario' doesn't exist
[Method: POST, Route: empleados]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:38:38 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:38:38 --> [Caused by] mysqli_sql_exception: Table 'telat.tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-06-18 20:39:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-18 20:41:39 --> mysqli_sql_exception: Table 'telat.tipo_usuario' doesn't exist in C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', Array, NULL, 'id_tipo_usuario')
#6 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', Array, Array, 'id_tipo_usuario')
#7 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\crud_telat\app\Controllers\Empleados.php(68): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Empleados->create()
#10 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#11 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\crud_telat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-06-18 20:41:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.tipo_usuario' doesn't exist
[Method: POST, Route: empleados]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:41:39 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:41:39 --> [Caused by] mysqli_sql_exception: Table 'telat.tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'id_tipo_usuario')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '1', [...], [...], 'id_tipo_usuario')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-06-18 20:41:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 20:42:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-18 20:42:14 --> ErrorException: Undefined array key "id_tipo_usuario"
[Method: GET, Route: empleados/new]
in APPPATH\Views\empleados\nuevo.php on line 23.
 1 APPPATH\Views\empleados\nuevo.php(23): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_tipo_usuario"', 'C:\\xampp\\htdocs\\crud_telat\\app\\Views\\empleados\\nuevo.php', 23)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\crud_telat\\app\\Views\\empleados\\nuevo.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('empleados/nuevo', [], true)
 5 APPPATH\Controllers\Empleados.php(43): view('empleados/nuevo', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->new()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-06-18 20:42:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-18 20:42:16 --> ErrorException: Undefined array key "id_tipo_usuario"
[Method: GET, Route: empleados/new]
in APPPATH\Views\empleados\nuevo.php on line 23.
 1 APPPATH\Views\empleados\nuevo.php(23): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_tipo_usuario"', 'C:\\xampp\\htdocs\\crud_telat\\app\\Views\\empleados\\nuevo.php', 23)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\crud_telat\\app\\Views\\empleados\\nuevo.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('empleados/nuevo', [], true)
 5 APPPATH\Controllers\Empleados.php(43): view('empleados/nuevo', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->new()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-06-18 20:42:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-18 20:42:20 --> ErrorException: Undefined array key "id_tipo_usuario"
[Method: GET, Route: empleados/new]
in APPPATH\Views\empleados\nuevo.php on line 23.
 1 APPPATH\Views\empleados\nuevo.php(23): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_tipo_usuario"', 'C:\\xampp\\htdocs\\crud_telat\\app\\Views\\empleados\\nuevo.php', 23)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\crud_telat\\app\\Views\\empleados\\nuevo.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('empleados/nuevo', [], true)
 5 APPPATH\Controllers\Empleados.php(43): view('empleados/nuevo', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->new()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-06-18 20:42:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 20:42:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 20:43:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-18 20:43:53 --> mysqli_sql_exception: Table 'telat.tipo_usuario' doesn't exist in C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', Array, NULL, 'tipo_usuario')
#6 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('tipo_usuario', 'tipo_usuario', '1', Array, Array, 'tipo_usuario')
#7 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\crud_telat\app\Controllers\Empleados.php(68): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Empleados->create()
#10 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#11 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\crud_telat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-06-18 20:43:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.tipo_usuario' doesn't exist
[Method: POST, Route: empleados]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'tipo_usuario')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('tipo_usuario', 'tipo_usuario', '1', [...], [...], 'tipo_usuario')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:43:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'tipo_usuario')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('tipo_usuario', 'tipo_usuario', '1', [...], [...], 'tipo_usuario')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:43:53 --> [Caused by] mysqli_sql_exception: Table 'telat.tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'tipo_usuario')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('tipo_usuario', 'tipo_usuario', '1', [...], [...], 'tipo_usuario')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-18 20:45:05 --> mysqli_sql_exception: Table 'telat.tipo_usuario' doesn't exist in C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', Array, NULL, 'tipo_usuario')
#6 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('tipo_usuario', 'tipo_usuario', '1', Array, Array, 'tipo_usuario')
#7 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\crud_telat\app\Controllers\Empleados.php(68): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Empleados->create()
#10 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#11 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\crud_telat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-06-18 20:45:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.tipo_usuario' doesn't exist
[Method: POST, Route: empleados]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'tipo_usuario')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('tipo_usuario', 'tipo_usuario', '1', [...], [...], 'tipo_usuario')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:45:05 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'tipo_usuario')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('tipo_usuario', 'tipo_usuario', '1', [...], [...], 'tipo_usuario')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:45:05 --> [Caused by] mysqli_sql_exception: Table 'telat.tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'tipo_usuario')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('tipo_usuario', 'tipo_usuario', '1', [...], [...], 'tipo_usuario')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-18 20:45:09 --> mysqli_sql_exception: Table 'telat.tipo_usuario' doesn't exist in C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', Array, NULL, 'tipo_usuario')
#6 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('tipo_usuario', 'tipo_usuario', '1', Array, Array, 'tipo_usuario')
#7 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\crud_telat\app\Controllers\Empleados.php(68): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Empleados->create()
#10 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#11 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\crud_telat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-06-18 20:45:09 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.tipo_usuario' doesn't exist
[Method: POST, Route: empleados]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'tipo_usuario')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('tipo_usuario', 'tipo_usuario', '1', [...], [...], 'tipo_usuario')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:45:09 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'tipo_usuario')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('tipo_usuario', 'tipo_usuario', '1', [...], [...], 'tipo_usuario')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:45:09 --> [Caused by] mysqli_sql_exception: Table 'telat.tipo_usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `tipo_usuario`
WHERE `id` = \'1\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tipo_usuario`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('1', 'id', [...], null, 'tipo_usuario')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('tipo_usuario', 'tipo_usuario', '1', [...], [...], 'tipo_usuario')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 20:47:11 --> CodeIgniter\Validation\Exceptions\ValidationException: "max_lenght" is not a valid rule.
[Method: POST, Route: empleados]
in SYSTEMPATH\Validation\Validation.php on line 345.
 1 SYSTEMPATH\Validation\Validation.php(345): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('max_lenght')
 2 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('clave', 'clave', '1234567888', [...], [...], 'clave')
 3 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 4 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-06-18 20:51:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 20:51:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 20:52:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 20:53:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 20:53:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 22:33:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-18 22:33:33 --> Error: Call to undefined function set_value()
[Method: GET, Route: empleados/new]
in APPPATH\Views\empleados\nuevo.php on line 31.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('empleados/nuevo', [], true)
 4 APPPATH\Controllers\Empleados.php(43): view('empleados/nuevo', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->new()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-06-18 22:36:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 22:37:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 22:38:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 22:38:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 22:41:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 22:42:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 22:49:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 22:50:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 22:55:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-18 22:55:04 --> ErrorException: Undefined array key "id_tipo_usuario"
[Method: GET, Route: empleados/new]
in APPPATH\Views\empleados\nuevo.php on line 23.
 1 APPPATH\Views\empleados\nuevo.php(23): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_tipo_usuario"', 'C:\\xampp\\htdocs\\crud_telat\\app\\Views\\empleados\\nuevo.php', 23)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\crud_telat\\app\\Views\\empleados\\nuevo.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('empleados/nuevo', [], true)
 5 APPPATH\Controllers\Empleados.php(44): view('empleados/nuevo', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->new()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-06-18 22:55:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 22:58:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 22:58:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 22:58:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 22:58:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:04:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:04:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:11:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:11:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:11:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:11:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:11:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:11:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:11:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:11:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:11:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:11:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:11:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:11:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:11:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:11:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:30:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:31:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:34:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-18 23:34:43 --> mysqli_sql_exception: Table 'telat.departamentos' doesn't exist in C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('3', 'id', Array, NULL, 'id_tipo_usuario')
#6 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '3', Array, Array, 'id_tipo_usuario')
#7 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\crud_telat\app\Controllers\Empleados.php(68): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Empleados->create()
#10 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#11 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\crud_telat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\crud_telat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-06-18 23:34:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.departamentos' doesn't exist
[Method: POST, Route: empleados]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `departamentos`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('3', 'id', [...], null, 'id_tipo_usuario')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '3', [...], [...], 'id_tipo_usuario')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 23:34:43 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'telat.departamentos' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `departamentos`
WHERE `id` = \'3\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `departamentos`
WHERE `id` = \'3\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `departamentos`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('3', 'id', [...], null, 'id_tipo_usuario')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '3', [...], [...], 'id_tipo_usuario')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-18 23:34:43 --> [Caused by] mysqli_sql_exception: Table 'telat.departamentos' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1
FROM `departamentos`
WHERE `id` = \'3\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `departamentos`
WHERE `id` = \'3\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `departamentos`
WHERE `id` = \'3\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `departamentos`
WHERE `id` = :id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(168): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_not_unique('3', 'id', [...], null, 'id_tipo_usuario')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('id_tipo_usuario', 'id_tipo_usuario', '3', [...], [...], 'id_tipo_usuario')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Empleados.php(68): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->create()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-06-18 23:35:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:35:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-18 23:35:02 --> ErrorException: Array to string conversion
[Method: GET, Route: empleados/new]
in APPPATH\Views\empleados\nuevo.php on line 9.
 1 APPPATH\Views\empleados\nuevo.php(9): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'C:\\xampp\\htdocs\\crud_telat\\app\\Views\\empleados\\nuevo.php', 9)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\crud_telat\\app\\Views\\empleados\\nuevo.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('empleados/nuevo', [], true)
 5 APPPATH\Controllers\Empleados.php(43): view('empleados/nuevo', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->new()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-06-18 23:38:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:38:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:38:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-18 23:38:45 --> ErrorException: Array to string conversion
[Method: GET, Route: empleados/new]
in APPPATH\Views\empleados\nuevo.php on line 9.
 1 APPPATH\Views\empleados\nuevo.php(9): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'C:\\xampp\\htdocs\\crud_telat\\app\\Views\\empleados\\nuevo.php', 9)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\crud_telat\\app\\Views\\empleados\\nuevo.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('empleados/nuevo', [], true)
 5 APPPATH\Controllers\Empleados.php(43): view('empleados/nuevo', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->new()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-06-18 23:40:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:40:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:40:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:41:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:41:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:41:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:41:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:41:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:43:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:43:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:43:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:48:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-18 23:48:41 --> Error: Call to undefined function set_value()
[Method: GET, Route: empleados/new]
in APPPATH\Views\empleados\nuevo.php on line 31.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('empleados/nuevo', [], true)
 4 APPPATH\Controllers\Empleados.php(43): view('empleados/nuevo', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Empleados->new()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-06-18 23:50:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:51:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:51:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:52:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:52:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:52:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:54:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:54:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-18 23:54:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
